### DISCUSSION TITLE
problem with style on my page

### DISCUSSION THREAD

**Discussion by gogi100** [rejetto/hfs#228] [General]
i created the part on my page that wider than the document display area, when i use monitor 24'', but the monitor with 19'' displays page ok. my page is
![mypage](https://user-images.githubusercontent.com/21057856/234183861-561b3a60-0e3e-4869-8da8-3969207015ee.jpg)

how to expand this part with buttons and documents like above the part?
i set my html custom
[custom.txt](https://github.com/rejetto/hfs/files/11319027/custom.txt)

**Comment by SanokKule** [comment]
The default width for the listing is `50em`
You might want to make your top part be `50em` instead of `65%`
If you want to just extend the listing, do this:
```
#root {
	max-width: your-value;
}
```
I would highly suggest against using the % values as it will look horrible on mobile.

**Comment by gogi100** [comment]
thanks
